CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.3.0](https://github.com/sonata-project/exporter/compare/2.2.0...2.3.0) - 2020-07-13
+### Added
+- [[#343](https://github.com/sonata-project/exporter/pull/343)] Added support for array and traversable in `DoctrineORMQuerySourceIterator`, `DoctrineODMQuerySourceIterator` and `PropelCollectionSourceIterator`. ([@VincentLanglet](https://github.com/VincentLanglet))
```

Comment:
Majority of previous messages are not using full stop (although some are).
I don't know if we are editing this or not, I'm just trying to keep consistency.
```suggestion
- [[#343](https://github.com/sonata-project/exporter/pull/343)] Added support for array and traversable in `DoctrineORMQuerySourceIterator`, `DoctrineODMQuerySourceIterator` and `PropelCollectionSourceIterator` ([@VincentLanglet](https://github.com/VincentLanglet))
```